### PR TITLE
Organization optimizations

### PR DESCRIPTION
**Pull request by gaspergrom**
# Changes proposed ✍️

### What
copilot:summary
​
copilot:poem

### How
copilot:walkthrough

## Checklist ✅
- [ ] Label appropriately with `Feature`, `Improvement`, or `Bug`.
- [ ] Add screenshots to the PR description for relevant FE changes
- [ ] New backend functionality has been unit-tested.
- [ ] API documentation has been updated (if necessary) (see [docs on API documentation](https://docs.crowd.dev/docs/updating-api-documentation)).
- [ ] [Quality standards](https://github.com/CrowdDotDev/crowd-github-test-public/blob/main/CONTRIBUTING.md#quality-standards) are met.

### CODE REVIEW

File: backend/src/database/repositories/member/memberOrganizationsRepository.ts [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+  }
+
+  static async delete(id: string, memberId: string, options: IRepositoryOptions) {
+    const transaction = await SequelizeRepository.createTransaction(options)
+    try {
+      const txOptions = { ...options, transaction }
+      const qx = SequelizeRepository.getQueryExecutor(txOptions, transaction)
+
+      // Delete organization
+      await deleteMemberOrganization(qx, memberId, id)
```
Comment (sausage-todd) [inline backend/src/database/repositories/member/memberOrganizationsRepository.ts:126]: You're executing only one query in this transaction, so you don't have to care about transactions in this function, unless you really want to

Comment (sausage-todd) [reply backend/src/database/repositories/member/memberOrganizationsRepository.ts:126]: Same applies to `update()` and `create()`

File: backend/src/database/repositories/member/memberOrganizationsRepository.ts [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+        }),
+        {},
+      )
+
+      // Format the results
+      const result = memberOrganizations.map((mo) => ({
+        ...(orgByid[mo.organizationId] || {}),
+        id: mo.organizationId,
+        memberOrganizations: mo,
+      }))
```
Comment (sausage-todd) [inline backend/src/database/repositories/member/memberOrganizationsRepository.ts:55]: Nice

**Review by sausage-todd — APPROVED** [review]

File: backend/src/database/repositories/member/memberOrganizationsRepository.ts (hunk 1 above)
Comment (gaspergrom) [reply backend/src/database/repositories/member/memberOrganizationsRepository.ts]: removed transactions wherever needed

**Pull request merged**